tests/foreman/api/test_subnet.py

```diff
@@ -32,119 +40,170 @@
+            subnet_parameters_attributes=parameter
+        ).create()
+        self.assertEqual(
+            subnet.subnet_parameters_attributes[0]['name'],
+            parameter[0]['name']
+        )
+        self.assertEqual(
+            subnet.subnet_parameters_attributes[0]['value'],
+            parameter[0]['value']
+        )
+
     @tier1
     def test_positive_add_parameter(self):
         """Parameters can be created in subnet
-
         :id: c1dae6f4-45b1-45db-8529-d7918e41a99b
-
         :steps:
-
             1. Create Subnet with all the details
             2. Create subnet parameter with single key and single value
-
         :expectedresults: The parameter should be created in subnet
         """
+        subnet = entities.Subnet().create()
+        for name in generate_strings_list(exclude_types=['html']):
+            with self.subTest(name):
+                value = gen_string('utf8')
+                subnet_param = entities.Parameter(
+                    subnet=subnet.id,
+                    name=name,
+                    value=value
+                ).create()
+                self.assertEqual(subnet_param.name, name)
+                self.assertEqual(subnet_param.value, value)
 
-    @stubbed()
     @tier1
     def test_positive_add_parameter_with_multiple_values(self):
         """Subnet parameters can be created with multiple values
-
         :id: b3de6f96-7c39-4c44-b91c-a6d141f5dd6a
-
         :steps:
-
             1. Create Subnet with all the details
             2. Create subnet parameter having single key and multiple values
                 separated with comma
-
         :expectedresults: The parameter with multiple values should be saved
             in subnet
         """
+        subnet = entities.Subnet().create()
+        name = gen_string('alpha')
+        values = ', '.join(generate_strings_list(exclude_types=['html']))
+        with self.subTest(name):
+            subnet_param = entities.Parameter(
+                name=name,
+                subnet=subnet.id,
+                value=values
+            ).create()
+            self.assertEqual(subnet_param.name, name)
+            self.assertEqual(subnet_param.value, values)
 
-    @stubbed()
     @tier1
     def test_positive_create_with_parameter_and_valid_separator(self):
         """Subnet parameters can be created with multiple names with valid
         separators
-
         :id: d1e2d75a-a1e8-4767-93f1-0bb1b75e10a0
-
         :steps:
-
             1. Create Subnet with all the details
             2. Create subnet parameter having key with multiple names separated
                 by valid separators(e.g fwd slash) and value
-
         :expectedresults: The parameter with multiple names separated by valid
             separators should be saved in subnet
         """
+        subnet = entities.Subnet().create()
+        valid_separators = [',', '/', '-', '|']
+        valid_names = []
+        for separator in valid_separators:
```

Comment:
same here.
Test case is valid from testing separator logic perspective, but not from word 'multiple'

Reply:
Ok if word multiple confuses a lot, let me 'remove' that word itself.